lib/src/browser/disposable_browser.dart

```diff
@@ -68,14 +69,14 @@ class _InnerDisposable extends disposable_common.Disposable {
 ///
 ///        MyDisposable() {
 ///          var thing = new ThingThatRequiresCleanup();
-///          manageDisposer(() {
+///          getManagedDisposer(() {
 ///            thing.cleanUp();
 ///            return new Future(() {});
 ///          });
 ///        }
 ///      }
 ///
-/// Cleanup will then be automatically performed when the containing
+/// Cleanup will then be automatically performed when the parent
```

Comment:
I went through and made sure that there was consistent use of `parent` vs `containing` object, not really sure which one I actually prefer. I chose `parent` simply because it was used more. From the perspective of the AST a managed object could definitely be thought of a child object, but `containing` maybe seems more natural for the disposable pattern. I'm on the fence.